core-web/libs/edit-content/src/lib/fields/dot-edit-content-category-field/components/dot-edit-content-category-field-dialog/dot-edit-content-category-field-dialog.component.scss

```diff
@@ -0,0 +1,37 @@
+@use "variables" as *;
+
+:host {
+    display: block;
+    height: 100%;
+
+    .category-field__dialog {
+        display: grid;
+        grid-template-columns: 2fr 1fr;
+        max-width: 1200px;
+        margin: auto;
+    }
+
+    .category-field__left-pane,
+    .category-field__right-pane {
+        padding: $spacing-3;
+        gap: $spacing-3;
+    }
+
+    .category-field__left-pane {
+        background-color: $color-palette-gray-300;
+    }
+
+    .category-field__selected-categories {
+        border: $field-border-size solid $color-palette-gray-400;
+        border-radius: $border-radius-sm;
+        padding: $spacing-3;
+    }
+
+    .category-field__actions {
+        gap: $spacing-1;
+    }
+}
+
+::ng-deep .category-field__dialog .p-dialog-content {
+    padding: 0;
+}
```

Comment:
This in the one that needs the nested with the `:host`.

Reply:
This is dangerous because it can affect other dialogs in the page, no?

Reply:
It will be deleted. In the draft of the change to use sidebar instead, I rid of those lines.